quickfixj-core/src/main/java/quickfix/Initiator.java

```diff
@@ -30,42 +30,42 @@
      *
      * @see quickfix.SessionFactory#SETTING_CONNECTION_TYPE
      */
-    public static final String SETTING_RECONNECT_INTERVAL = "ReconnectInterval";
+    String SETTING_RECONNECT_INTERVAL = "ReconnectInterval";
```

Comment:
IntelliJ says that modifiers at interfaces are redundant, in an interface they are always static final for constants.